### ISSUE TITLE
Browser specific pseudo-elements error when used in other browsers

### ISSUE THREAD

**Issue by hadihallak** [modulz/stitches#64]
Fixing it is probably as simple as wrapping the insertRule call in a try-catch block to avoid the browser complaining about an unsupported selector.

**Error message:**
```js
SyntaxError: Failed to execute 'insertRule' on 'CSSStyleSheet': Failed to parse the rule '.bc_0::-moz-range-track{background-color:transparent;}'.
toString
src/index.ts:65
  62 |     cssRule = `.${className[0]}{${this.cssHyphenProp}:${value};}`;
  63 |   }
  64 | 
> 65 |   sheets[this.screen].insertRule(
     | ^  66 |     this.screen ? screens[this.screen](cssRule) : cssRule
  67 |   );
  68 | }
```

**the following code will give the above error when used in chrome:**

```tsx

import React from 'react'

import { createStyled } from '@stitches/styled'

export const { styled, css } = createStyled({})

export const Slider = styled.div({
  '::-moz-range-track': { backgroundColor: 'transparent' },
})

function App() {
  return (
    <div className="App">
      <Slider> Working </Slider>
    </div>
  )
}

export default App
```

**Comment by christianalfoni** [comment]
I believe you could also use this:

https://github.com/modulz/stitches/blob/master/packages/css/src/utils.ts#L281

Here you get a list of all the supported properties of the browser, so could check if it exists there. But yeah, then you would have to do that for every single property which would give some overhead. In support of the try catch here, but just wanted to mention it 😄

**Comment by hadihallak** [comment]
fixed in #120
